bugbug/model.py

```diff
@@ -52,6 +55,9 @@ def get_important_features(self, cutoff, shap_values):
 
         return top_features
 
+    def calculate_labels(self, classes, bugs):
+        return np.array([classes[bug['id']] for bug in bugs()])
```

Comment:
We could make each model directly return the labels in the format they want in `get_labels`

Reply:
Can we tackle this in a different PR, since it involved changes to alla the models we currently have.

Reply:
Can we tackle this in a different PR since it involves changes in all the rest models as well ?

Reply:
OK!